bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -116,13 +116,14 @@ class Poll extends Component {
   }
 
   componentDidUpdate() {
-    const { currentUser } = this.props;
+    const { amIPresenter } = this.props;
 
     if (Session.equals('resetPollPanel', true)) {
       this.handleBackClick();
     }
 
-    if (!currentUser.presenter) {
+    console.error(`amIPresenter `, amIPresenter)
```

Comment:
I think this is a left over debug log.

Reply:
Looking at my code, it was removed before pushing. Maybe GitHub is losing it? oO